Trappist/src/Promact.Trappist.Web/wwwroot/app/core/http.service.ts

```diff
@@ -11,7 +11,13 @@ import 'rxjs/add/operator/toPromise';
 
 export class HttpService {
 
-    constructor(private http: Http) {
+    constructor(private http: Http, defaultOptions: RequestOptions) {
+
+        //Prevent request caching for internet explorer
+        defaultOptions.headers.append('Cache-control', 'no-cache');
```

Comment:
there is no connection between bootstrap and other css resources and request being made by http.service.ts. http.service.ts is only for API requests